### ISSUE TITLE
I have 3 tuners set but it only uses 1

### ISSUE THREAD

**Issue by whoathere** [xteve-project/xTeVe#90]
#### Please first read the [documentation](https://github.com/xteve-project/xTeVe-Wiki/blob/master/en/configuration.md). Problems affecting the client, e.g. Plex or Emby should be reported there in the forum.

**Describe the bug**
I have XTeVe set for 3 tuners because I have 3 connections to the IPTV service I am using, but when I try to make a second connection I get an xTeVe banner that says "xTeVe No more streams available", which is not true. I can open up additional streams using the M3U file I have and VLC. I am using Plex to connect to xTeVe so in Plex I am seeing the xTeVe banner saying "No more streams available".

**Server (please complete the following information):**
- OS Linux - Ubuntu 16.04
- Arch Intel 64 Bit
- xTeVe Version and Build [to be found in the xTeVe log]
Version:               2.1.1 Build: 0112
Database Version:      2.1.0
Plex says under the Tuner for xTeVe that theres 3 tuners.

**To Reproduce**
1. In Plex, Go to 'LiveTV & DVR'
2.  Click on "Guide" at the top
3. Scroll down to 'Literally any channel' and play the channel.
4. I get the banner "xTeVe No more streams available"
5. Please note that 1 connection is currently playing a channel. I can validate that I can stream a second channel by opening my M3U file with VLC and playing a second channel on it. I have 3 channels so I should be able to stream a 3rd channel in Plex.

**Expected behavior**
I should be able to stream a second and third channel through Plex.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Ive used the Plex Media Player as well as Chrome, Opera, Firefox and MSIE
 - Version Every browser is the current version

**Additional context**
Nothing additional to add

**Comment by mar-mei** [comment]
How many tuners have you set in the playlist?

![image](https://user-images.githubusercontent.com/43099631/74186144-570d4600-4c4a-11ea-956c-c85ba144b8b9.png)

**Comment by whoathere** [comment]
3
![tuners](https://user-images.githubusercontent.com/24444421/74186371-1c86b780-4c10-11ea-8557-17d3392b292e.jpg)

**Comment by whoathere** [comment]
Wait... You asked about playlist. So I went to playlist and it said 1... Ok... My bad... No bug and Im not sure how that changed...

I appreciate the help!

**Comment by whoathere** [comment]
Ok, so now when I try to play a second stream it ends both streams, where the first stream from Plex says "Live TV playback has ended." and it didnt. The second stream doesnt start and I get from Plex "Playback Error" - "Source is unavailable".

**Comment by whoathere** [comment]
To further troubleshoot, I decided to use a different browser. I have the Plex Media Player playing a station and through a web page I am trying to launch the second connection. In Chrome, it keeps failing, but I just tried FireFox and I am now playing a second station through Plex. I tried Opera and I was able to stream a second station as well. I also tried ROKU/Plex and it worked there as well just fine.

So, I guess this is a bug, but with a browser?
